docs/source/traits_api_reference/trait_handlers.rst

```diff
@@ -45,3 +39,10 @@ and may be removed in a future version of Traits.
 .. autodata:: TraitList
 
 .. autodata:: TraitTuple
+
+.. autoclass:: TraitPrefixList
```

Comment:
We need a new `.. deprecated` note here, since we're deprecating these in version `6.1`, not `6.0`.

Reply:
Hmm. Not sure why we had `autodata` rather than `autoclass` for the preceding entries. That should be fixed (but not necessarily in this PR).

Reply:
`autodata` also mentions the module name, don't think its needed, I went ahead and changed it to `autoclass`